getting-started/quick-start.md

```diff
@@ -0,0 +1,60 @@
+# Quick Start
+
+Axon provides a _**Quick Start Toolkit**_ to familarize yourself with the setup required for Axon Framework and Axon Server SE \(Standard Edition\). 
+
+The only pre-requisite to run the Quick Start is to have a Java 8+ JRE in your system.
+
+### Quick Start Toolkit Download
+
+The Quick Start Toolkit package is available for download at the following location -&gt; [https://axoniq.io/download](https://axoniq.io/download)
+
+This package contains,
+
+* Axon Framework Binaries
+* Axon Server Standard Edition
+* Gift card sample application - Demo application designed to show various aspects of the platform
+* Getting started guide
+
+### Running the Quick Start \(Java 9+\)
+
+1. Unzip `AxonQuickStart-VERSION.zip`
+2. Run the [Axon Server](../axon-server-introduction.md): `$ java -jar AxonServer/axonserver-VERSION.jar`
+3. Axon Server web dashboard should be available here [`http://localhost:8024/`](http://localhost:8024/)
+4. Run the demo application: `$ cd giftcard-demo && ./mvnw spring-boot:run`
+5. Demo application should be available here [`http://localhost:8080/`](http://localhost:8080/)
+6. Explore the `README.md`
+
+### Running the Quick Start on Java 8
+
+Quick Start Demo app can run on Java 8 with a small change in the `pom.xml` file
```

Comment:
Ok, I'll throw this in: Why are we still spending time on Java 8 JDK? Java 11 on compatibility setting for Java 8 I can relate to, but the 1.8 JDK should not be used.

Reply:
I'd argue because Axon Framework and the extensions are still build with Java 8. Once we move to 5.x, for which also definitely recommend to move to 11, then I think it time to change this stuff too.